spec.html

```diff
@@ -36061,21 +36061,21 @@ <h1>Runtime Semantics: CompileCharacterClass</h1>
           <dt>description</dt>
           <dd>It returns a Record with fields [[CharSet]] (a CharSet) and [[Invert]] (a Boolean).</dd>
         </dl>
-        <emu-grammar>CharacterClass :: `[` ClassRanges `]`</emu-grammar>
+        <emu-grammar>CharacterClass :: `[` ClassContents `]`</emu-grammar>
         <emu-alg>
-          1. Let _A_ be CompileToCharSet of |ClassRanges|.
+          1. Let _A_ be CompileToCharSet of |ClassContents|.
           1. Return the Record { [[CharSet]]: _A_, [[Invert]]: *false* }.
         </emu-alg>
-        <emu-grammar>CharacterClass :: `[` `^` ClassRanges `]`</emu-grammar>
+        <emu-grammar>CharacterClass :: `[` `^` ClassContents `]`</emu-grammar>
         <emu-alg>
-          1. Let _A_ be CompileToCharSet of |ClassRanges|.
+          1. Let _A_ be CompileToCharSet of |ClassContents|.
           1. If _UnicodeSets_ is *true*, then
             1. Return the Record { [[CharSet]]: CharacterComplement(_A_), [[Invert]]: *false* }.
           1. Return the Record { [[CharSet]]: _A_, [[Invert]]: *true* }.
         </emu-alg>
       </emu-clause>
 
-      <emu-clause id="sec-compiletocharset" type="sdo" oldids="sec-classranges,sec-nonemptyclassranges,sec-nonemptyclassrangesnodash,sec-classatom,sec-classatomnodash,sec-classescape,sec-characterclassescape">
+      <emu-clause id="sec-compiletocharset" type="sdo" oldids="sec-ClassContents,sec-nonemptyclassranges,sec-nonemptyclassrangesnodash,sec-classatom,sec-classatomnodash,sec-classescape,sec-characterclassescape">
```

Comment:
preserve lowercase?

Reply:
Good catch. This line actually shouldn’t be changed at all.

```suggestion
      <emu-clause id="sec-compiletocharset" type="sdo" oldids="sec-classranges,sec-nonemptyclassranges,sec-nonemptyclassrangesnodash,sec-classatom,sec-classatomnodash,sec-classescape,sec-characterclassescape">
```